mifospay/src/main/res/values/styles_linear.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="LinearLayout"/>
+
+    <style name="LinearLayout.Base">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">match_parent</item>
+        <item name="android:orientation">vertical</item>
+    </style>
+
+    <style name="LinearLayout.Wrap">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+    </style>
+
+    <style name="LinearLayout.Width">
```

Comment:
I think it is much better to define layout width and height in the layout itself, not in the style. Even by that are you sure `name="LinearLayout.Width"` is a general and a good name for your defined style which everyone can understand?

Reply:
I think, it would be a global style which many would use in the future whenever they need the linear layout